mkdocs.yml

```diff
@@ -74,7 +74,7 @@ plugins:
             docstring_style: numpy
             filters:
               - "!^_"
-              - "__.+"
```

Comment:
These were added at the request of a user in https://github.com/ibis-project/ibis/issues/5424.

Reply:
Maybe we can put them behind a toggle or something, but I don't think they should be on by default

Reply:
SGTM!